README.md

```diff
@@ -92,4 +92,21 @@ When the playbook runs, it will do the following tasks:
 - Copy todoapp.service file
 - Run todo app
 
-When these steps are complete, you should be able to navigate to the application from a brower. Just enter the IP address of the host into your address bar, followed by :5000. For example http://13.41.161.109:5000/.
\ No newline at end of file
+When these steps are complete, you should be able to navigate to the application from a brower. Just enter the IP address of the host into your address bar, followed by :5000. For example http://13.41.161.109:5000/.
+
+## Building and running docker containers
+To build and run a development docker container, run the following commands in a docker prompt:
+```bash
+docker build --target development --tag todo-app:dev .
+docker run -d -p 7000:8000 --env-file .\.env --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:dev
```

Comment:
Since you're not working in a particular work directory, I don't think your bind mount quite lines up:
```suggestion
docker run -d -p 7000:8000 --env-file .\.env --mount type=bind,source="$(pwd)"/todo_app,target=/todo_app todo-app:dev
```
This means that if you make changes to your code (e.g. to `app.py`) after you've started a container, I don't see those being automatically picked up currently